CHANGELOG.md

```diff
@@ -2,7 +2,12 @@
 Running changelog of releases since `2.0.0-rc.4`
 
 ## v2.0.0
-- Official Release of v2 Okta Golang SDK
+- Official Release of v2 Okta Golang SDK (June 4, 2020)
+- `CSR` Model has been renamed to `Csr`
```

Comment:
Aren't Csr entities new to this round?

Reply:
Oh! I think I get it. You are comparing it against preview RCs, right?

Reply:
Yup